test/podman_pause.bats

```diff
@@ -110,3 +109,32 @@ function teardown() {
     echo "$output"
     [ "$status" -eq 0 ]
 }
+
+@test "pause container by id and name" {
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} run -d $BB sleep 60
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} ps  --format "{{.Names}}"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_name=$output
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} run -d $BB sleep 60
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id=`echo "$output" | tail -n 1`
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} pause $ctr_id
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} pause $ctr_name
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} unpause $ctr_id
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} unpause $ctr_name
+    echo "$output"
+    [ "$status" -eq 0 ]
+    run ${PODMAN_BINARY} ${PODMAN_OPTIONS} rm $ctr_id
```

Comment:
Probably should just be `rm -a` to clear both containers

Reply:
that's probably a better test, good thought.